aws-iam-group-assume-role/main.tf

```diff
@@ -32,3 +34,22 @@ resource "aws_iam_group_policy_attachment" "assume-role" {
   policy_arn = "${aws_iam_policy.assume-role.arn}"
   group      = "${aws_iam_group.assume-role.name}"
 }
+
+# # https://github.com/hashicorp/terraform/issues/1178
+
+# # Pseudo dependencies to ensure dependencies actually get enforced
```

Comment:
why are these commented out?

Reply:
Although this workaround as described in the comments on the above issue is actually semantically correct, in practice the need to shell out makes Terraform complain about running out of file handles. I wanted something in here that shows that a) We currently do this incorrectly which is why we have to double apply whenever we add a new user, and b) here's semantically what we want it to do.

Reply:
maybe add a comment to that effect in the code? otherwise having that code commented out there doesn't give us much

Reply:
It's in the comment of the commit, not in the code itself. Good enough?

Reply:
Actually added the comment to the code, since I realize the ensuing squash will remove it.